bounties/level-2/video-tutorials.md

```diff
@@ -0,0 +1,55 @@
+# Video tutorials
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 750 XLM (+750 XLM for established creators) | 1000 XLM | Continuous 
```

Comment:
I think creating high-quality appealing content is way more effort then judging the result. So even the basic hunter's bounty should be higher then the reward.

Reply:
Agreed. I think the reviewer should need to watch the tutorial and spot any issues with it. Maybe we can increase the hunter award and reduce reviewer's.